src/testdir/test_cmdline.vim

```diff
@@ -2621,6 +2621,10 @@ func Test_cmd_map_cmdlineChanged()
   call feedkeys(":\<F1>\<CR>", 'xt')
   call assert_equal(['a', 'ab'], g:log)
 
+  let g:log = []
+  call feedkeys(":foo\<Left>\<Left>\<Del>\<Del>\<Esc>", 'xt')
+  call assert_equal(['f', 'fo', 'foo', 'fo', 'f'], g:log)
+
```

Comment:
This test is for `CmdlineChanged` interaction with `<Cmd>` mappings. It's better to add this in `Test_cmdline_changed()`.

Reply:
Thanks.  I have added new function.  Because current `Test_cmdline_changed()` does not check full command line.